owenhallgren/whats-cookin-starter-kit
# Feature/refactor

## Is this a fix or a feature?
Feature
## What is the change?
This adds styling corrections, test corrections, and refactoring all of the code
## What does it fix?
## Where should the reviewer start?
The reviewer should start by running the linter
## How should this be tested?
Run over every file with the linter --- check for violations not including variable definitions, line lengths, and data indentation errors
